src/apps/backend/modules/config/config_service.py

```diff
@@ -1,22 +1,34 @@
-from typing import Generic,TypeVar, cast
+from typing import Generic, Optional, cast
+
 from modules.common.types import ErrorCode
+from modules.config.internals.config_manager import ConfigManager
+from modules.config.types import T
 from modules.error.custom_errors import MissingKeyError
-from modules.config.internals.config import Config
 
-T = TypeVar('T')
 
 class ConfigService(Generic[T]):
-    @staticmethod
-    def load_config() -> None:
-        Config.load_config()
+    config_manager: ConfigManager = ConfigManager()
+
+    @classmethod
+    def load_config(cls) -> None:
+        """
+        Load the configuration files
```

Comment:
Comments like this are not good, see [How to write better code comments](https://dev.to/adammc331/todo-write-a-better-comment-4c8c), a reference mentioned in our [handbook](https://github.com/jalantechnologies/handbook/blob/main/engineering/index.md)

Please audit rest of the changes in the PR with this point of view and address them as well.

Reply:
Why is this class method and not static method (unlike most of our other classes)?